tests/ctl/core/test_api.py

```diff
@@ -733,6 +850,49 @@ async def test_system_create_invalid_privacy_declarations(
             len(PrivacyDeclaration.all(db)) == 0
         )  # ensure neither of our declarations were created
 
+    async def test_system_create_invalid_legal_basis_for_profiling(
+        self, generate_auth_header, test_config, system_create_request_body
+    ):
+        system_create_request_body.legal_basis_for_profiling = "bad_basis"
+        auth_header = generate_auth_header(scopes=[SYSTEM_CREATE])
+
+        result = _api.create(
+            url=test_config.cli.server_url,
+            headers=auth_header,
+            resource_type="system",
+            json_resource=system_create_request_body.json(exclude_none=True),
+        )
+
+        assert result.status_code == HTTP_422_UNPROCESSABLE_ENTITY
+        assert result.json()["detail"][0]["loc"] == [
+            "body",
+            "legal_basis_for_profiling",
+        ]
+
+
+@pytest.mark.unit
+class TestSystemGet:
+    def test_data_stewards_included_in_response(
+        self, test_config, system, system_manager
+    ):
+        result = _api.get(
+            url=test_config.cli.server_url,
+            headers=test_config.user.auth_header,
+            resource_type="system",
+            resource_id=system.fides_key,
+        )
+        assert result.status_code == 200
+        assert result.json()["fides_key"] == system.fides_key
+
+        data_stewards = result.json()["data_stewards"]
+        assert len(data_stewards) == 1
+        steward = data_stewards[0]
+
+        assert steward["id"] == system_manager.id
+        assert steward["username"] == system_manager.username
+        assert steward["first_name"] is None
```

Comment:
nit:

```suggestion
        assert not steward["first_name"]
```

Reply:
the main thing I'm trying highlight here though is just that the first name and last name are in this response, they just happen to be null for the data steward. I'll rework with that intent!